java/debugger/impl/src/com/intellij/debugger/jdi/MethodBytecodeUtil.java

```diff
@@ -292,6 +292,11 @@ public void visitMethodInsn(int opcode, String owner, String name, String desc,
       return locations;
     }
 
+    // Android Studio: 'getConstantPool' is not supported on android. (b/123863053)
```

Comment:
There shouldn't be an “Android Studio” comment here.

Reply:
Also “b/123863053” is a Google-internal URL that shouldn't be here.

Reply:
Thanks, I removed the comment